java/collectl/README

```diff
@@ -60,7 +60,7 @@ You can install it by running:
 
 Download the Kudu MR jar and run Spark with it on the classpath:
 
-    wget https://repository.cloudera.com/artifactory/cloudera-repos/org/apache/kudu/kudu-spark_2.10/0.10.0/kudu-spark_2.10-0.10.0.jar
+    wget https://repository.cloudera.com/artifactory/cloudera-repos/org/apache/kudu/kudu-spark_2.10/1.0.0/kudu-spark_2.10-1.0.0.jar
```

Comment:
We should use the Apache repo now:

`https://repository.apache.org/service/local/repositories/releases/content/org/apache/kudu/kudu-spark_2.10/1.0.1/kudu-spark_2.10-1.0.1.jar`